gammapy/irf/psf_analytical.py

```diff
@@ -354,3 +358,46 @@ def info(self, fractions=[0.68, 0.95], energies=Quantity([1., 10.], 'TeV'),
                            "E = {2:4.1f}: {3:5.8f}\n"
                            "".format(100 * fraction, theta, energy, radius))
         return ss
+
+
+
+    def to_table_psf(self, theta=None, offset_max=None, binsz=0.01, exposure=None):
```

Comment:
@adonath - From the parameter description it's not clear what `theta`, `offset_max` and `binsz` are ... at least I had to read all the code to understand how you use them.

Here's my suggestions:
- Put this as the description of `theta`: "Offset in the field of view"
- Remove the `offset_max` and `binsz` parameter. Instead put one `offset` parameter, which is the `offset` grid at which the PSF is sampled and that is passed to EnergyDependentTablePSF. You could make an example showing people how to call [gammapy.utils.axis.sqrt_space](http://docs.gammapy.org/en/latest/api/gammapy.utils.axis.sqrt_space.html) before to create a good `offset` array. I'm not sure if this should have a default ... you could put 1 deg which would work for all IACTs as far as I know.
- Add an `energy` parameter with default None and energy grid taken from self, as you do now. But if the user wants to sample with some other energy grid, then they have the option.

There are alternatives like making this a factory method on the EnergyDependentTablePSF class, or passing a EnergyDependentTablePSF object with pre-defined grid into this method, that have advantages when it comes to duplicating axis grid specs (like the log energy grid and sqrt offset grid). But they require more changes, also to the EnergyDependentTablePSF constructor, and aren't obviously better to me. So my suggestion would be to stick with what you have here, just to improve the parameters a bit like desribed above.